.circleci/config.yml

```diff
@@ -260,7 +260,7 @@ jobs:
       - run:
           name: "Build"
           command: |
-            make debug NUM_THREADS=16 MAX_HIGH_MEM_JOBS=8 MAX_LINK_JOBS=8 EXTRA_CMAKE_FLAGS="-DVELOX_ENABLE_ARROW=ON"
+            make debug NUM_THREADS=16 MAX_HIGH_MEM_JOBS=8 MAX_LINK_JOBS=8 EXTRA_CMAKE_FLAGS="-DVELOX_ENABLE_ARROW=ON -DVELOX_ENABLE_PARQUET=ON"
```

Comment:
Unintended change. Please, remove.

Reply:
I add it because I add a velox_dwio_parquet_reader for velox_exec_test.

Reply:
You should not do that. The problem here is not specific to file format, hence, can be reproduced with a DWRF file without introducing a dependency on Parquet.

Reply:
Thanks, I will use DWRF file.

Reply:
Thanks. Make sure to write the test case to auto-generate the file, don't check in the file itself. Looks for other tests in TableScanTest for examples.